Added Signposts, FAQs, and Ko-fi links (#21)
mcqn/my-first-pcb: README.md

# My First PCB 🦌

_Design a printed circuit board to get you started in the world of electronics design and small scale manufacturing!_

> ## Why PCB's? 🤔

Reviewer: ```suggestion
## Why PCBs? 🤔
```

* Maybe you’ve been building up your own circuits on a breadboard for a while now, and maybe soldering some of them onto strip-board, but want to make them look nicer.
* You want things to be more robust, professional and safe to help with product development or your own systems.

> * You’ve seen designs on the internet and need help to make it or adapt it yourself.

Reviewer: Pedantic old-school Internet person here :wink: the Internet is capitalised to differentiate it from other internetworked systems running IP but not connected to the Internet :grin: 
```suggestion
* You’ve seen designs on the Internet and need help to make it or adapt it yourself.
```

* You need to design your own printed circuit boards!

It’s easier than you think!

> This course is built to give you an understanding of the basic fundamentals in electronics production with KiCad using MCQN Ltd’s experience designing and developing products, systems and the Internet of Things. Everything from cuff-links, dust sensors, Museum installations and internet connected bells, LED drivers and bubble machines.

Reviewer: Always be pimping the main website, right? :grin: 
```suggestion
This course is built to give you an understanding of the basic fundamentals in electronics production with KiCad using [MCQN Ltd](https://mcqn.com)’s experience designing and developing products, systems and the Internet of Things. Everything from cuff-links, air quality sensors, museum installations and Internet connected bells, LED drivers and bubble machines.
```

Start off by running through the basics in [Designing my first #BadgeLife](DesigningMyFirstBadgeLife) and then choose your own route through the remaining tutorials based on what you're looking to build.

> If you benefit from this guide and would like to support us in our work on open source hardware, software, and guides like this one, consider [contributing via ko-fi]()https://ko-fi.com/s/d90aba90e1).

Reviewer: Hit me baby one more time :grin:
```suggestion
If you benefit from this guide and would like to support us in our work on open source hardware, software, and guides like this one, consider [contributing via ko-fi](https://ko-fi.com/s/d90aba90e1).
```
